### PR TITLE
[W5.4][T17-A1] Eric Loh

### PR DESCRIPTION
**Pull request by EricLoh**
Find and list people in address book by their tag (case sensitive)

### CODE REVIEW

**Comment by stan789** [comment]
Can include FindTagCommand in ParserTest Class by using JUnit Test to test.

File: src/seedu/addressbook/commands/FindtagCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.*;
+
+import static seedu.addressbook.commands.Command.getMessageForPersonListShownSummary;
+
+public class FindtagCommand extends Command{
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindtagCommand.java]: Missing header comment. All classes should have javadoc format header comments describing its purpose.

File: src/seedu/addressbook/commands/FindtagCommand.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     * @return list of persons found
+     */
+    private List<ReadOnlyPerson> getPersonsWithNameContainingTag(String tag) {
+        final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        for (ReadOnlyPerson person : addressBook.getAllPersons()) {
+           Iterator<Tag> iterator = person.getTags().iterator();
+           while(iterator.hasNext()){
+               if(iterator.next().tagName.equals(tag)){
+                   matchedPersons.add(person);
+                   break;
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindtagCommand.java]: **Arrowhead type code**
Refer to this [handout](http://www.comp.nus.edu.sg/~cs2103/AY1617S1/files/handouts/%5bL2P1%5d%20Good%20Code,%20Bad%20Code%20-%20Toward%20Production%20Quality%20Code.pdf) to find what arrowhead style is and how to eliminate it.

File: src/seedu/addressbook/commands/FindtagCommand.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+               }
+           }
+
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindtagCommand.java]: Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

**Comment by EricLoh** [comment]
My apologies, I have uploaded the JUnit Tests and the User Guide. Thanks!

**Comment by brandonyeoxg** [comment]
@EricLoh Hey Eric, you have fulfilled the requirements ^^ please close this PR.

**Pull request closed without merging**